tests/functional/user-data-manager.test.js

```diff
@@ -1,13 +1,14 @@
 /* eslint-env jest */
 /* eslint-disable no-unused-vars */
-import axios from 'axios'
-import { Constants, WordItem } from 'alpheios-data-models'
+import 'whatwg-fetch'
+import { ClientAdapters } from 'alpheios-client-adapters'
+import { Constants, WordItem, TextQuoteSelector, LanguageModelFactory as LMF } from 'alpheios-data-models'
 import UserDataManager from '@/controllers/user-data-manager'
 
 import IndexedDB from 'fake-indexeddb'
 import IDBKeyRange from 'fake-indexeddb/lib/FDBKeyRange'
 
-describe('indexeddb-workflow.test.js', () => {
+describe('user-data-manager.test.js', () => {
```

Comment:
if we're going to use the real remote service and not a mock here, then maybe we should be sure to clear the remote database for the test user before/after each test?

Reply:
Agree, I will add it (I have added clearing to remoteDBAdapter test, but missed it here)!

Reply:
Fixed